spec/features/index_page_spec.rb

```diff
@@ -297,6 +297,13 @@
       )
       expect(page).to have_css('h1.table-header', exact_text: 'Showing 101 to 102 of 102 results from org')
     end
+
+    it 'has GTM data attributes' do
+      total_results = page.find('span[data-gtm-pagination-total-results]')['data-gtm-pagination-total-results']
+
+      expect(page).to have_css('span[data-gtm-pagination-total-results]')
```

Comment:
Will the test error or fail if `span[data-gtm-pagination-total-results]` can't be found? As `.find` is being called before the expectation? Maybe just move the first line closer to where it's used?

Reply:
Good spot, thanks! I was overeager to keep the expectations together. Fixed